change/@fluentui-web-components-3b549149-2c12-435b-b925-e79f4fb05730.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31468&commitId=5869a8f737aec2ab6243764aad2d298ff858872e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Size.normal.chromium.png|2| Changed|
|Accordion.Size - RTL.normal.chromium.png|2| Changed|
|Accordion.Size - Dark Mode.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Avatar</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Active Appearance - Dark Mode.normal.chromium.png|2| Changed|
|Avatar.Active.normal.chromium.png|5| Changed|
|Avatar.Color - RTL.normal.chromium.png|1| Changed|
|Avatar.Active Appearance.normal.chromium.png|10| Changed|
|Avatar.Color.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Badge</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge.Appearance - Dark Mode.normal.chromium.png|15| Changed|
|Badge.Shape.normal.chromium.png|1| Changed|
|Badge.Appearance.normal.chromium.png|1| Changed|
|Badge.Default.normal.chromium.png|1| Changed|
|Badge.Size.normal.chromium.png|6| Changed|
|Badge.Color.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Button</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Large Icon Only - Dark Mode.default.chromium.png|4| Changed|
|Button.Large Icon Only - Dark Mode.pressed.chromium.png|5| Changed|
|Button.Large Icon Only.default.chromium.png|5| Changed|
|Button.Large With Icon - Dark Mode.pressed.chromium.png|5| Changed|
|Button.Large Icon Only.pressed.chromium.png|3| Changed|
|Button.Large With Icon - RTL.hover.chromium.png|4| Changed|
|Button.Large With Icon - Dark Mode.default.chromium.png|4| Changed|
|Button.Large Icon Only.hover.chromium.png|4| Changed|
|Button.Large With Icon.pressed.chromium.png|3| Changed|
|Button.Large With Icon - RTL.default.chromium.png|5| Changed|
|Button.Large Icon Only - Dark Mode.hover.chromium.png|3| Changed|
|Button.Large With Icon.default.chromium.png|5| Changed|
|Button.Large With Icon - RTL.pressed.chromium.png|3| Changed|
|Button.Large With Icon.hover.chromium.png|4| Changed|
|Button.Large With Icon - Dark Mode.hover.chromium.png|3| Changed|

</details>


  <details><summary><b>Checkbox</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Circular Checked.normal.chromium.png|5510| Changed|
|Checkbox.Default - RTL.normal.chromium.png|5393| Changed|
|Checkbox.Circular - RTL.normal.chromium.png|5383| Changed|
|Checkbox.Circular Disabled Checked - Dark Mode.normal.chromium.png|1310| Changed|
|Checkbox.Circular - Dark Mode.normal.chromium.png|976| Changed|
|Checkbox.Default.normal.chromium.png|5391| Changed|
|Checkbox.Circular Indeterminate.normal.chromium.png|5606| Changed|
|Checkbox.Disabled Checked - Dark Mode.normal.chromium.png|1155| Changed|
|Checkbox.Indeterminate - Dark Mode.indeterminate.chromium.png|1274| Changed|
|Checkbox.Disabled.normal.chromium.png|5439| Changed|
|Checkbox.Disabled - Dark Mode.normal.chromium.png|971| Changed|
|Checkbox.Disabled Checked.normal.chromium.png|5584| Changed|
|Checkbox.Indeterminate - Dark Mode.normal.chromium.png|1169| Changed|
|Checkbox.Indeterminate.indeterminate.chromium.png|5481| Changed|
|Checkbox.Circular Indeterminate - Dark Mode.indeterminate.chromium.png|1506| Changed|
|Checkbox.Indeterminate.normal.chromium.png|5492| Changed|
|Checkbox.Circular Disabled Checked.normal.chromium.png|5710| Changed|
|Checkbox.Circular Checked - Dark Mode.normal.chromium.png|1366| Changed|
|Checkbox.Checked - Dark Mode.normal.chromium.png|1136| Changed|
|Checkbox.Circular.normal.chromium.png|5384| Changed|
|Checkbox.Checked.normal.chromium.png|5388| Changed|
|Checkbox.Default - Dark Mode.normal.chromium.png|995| Changed|
|Checkbox.Circular Disabled.normal.chromium.png|5564| Changed|
|Checkbox.Circular Indeterminate - Dark Mode.normal.chromium.png|1359| Changed|
|Checkbox.Circular Disabled - Dark Mode.normal.chromium.png|1126| Changed|
|Checkbox.Circular Indeterminate.indeterminate.chromium.png|5589| Changed|

</details>


  <details><summary><b>Divider</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.With Svg.normal.chromium.png|3| Changed|
|Divider.With Svg - RTL.normal.chromium.png|3| Changed|
|Divider.Vertical With Svg.normal.chromium.png|4| Changed|

</details>


  <details><summary><b>Label</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Required - RTL.normal.chromium.png|11| Changed|
|Label.Size Small.normal.chromium.png|661| Changed|

</details>


  <details><summary><b>MenuList</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuList.Checkbox With Icons - RTL.normal.chromium.png|4| Changed|
|MenuList.Checkbox With Icons.1st selected.chromium.png|4| Changed|
|MenuList.Checkbox With Icons.2nd selected.chromium.png|5| Changed|
|MenuList.Checkbox - RTL.2nd selected.chromium.png|1| Changed|
|MenuList.Checkbox With Icons - RTL.1st selected.chromium.png|4| Changed|
|MenuList.Checkbox With Icons - RTL.2nd selected.chromium.png|5| Changed|
|MenuList.Checkbox With Icons.normal.chromium.png|4| Changed|
|MenuList.Radio With Icons.normal.chromium.png|4| Changed|
|MenuList.Radio With Icons.1st selected.chromium.png|4| Changed|
|MenuList.Radio With Icons.2nd selected.chromium.png|4| Changed|
|MenuList.With Icons.normal.chromium.png|4| Changed|
|MenuList.Radio With Icons - RTL.normal.chromium.png|4| Changed|
|MenuList.Checkbox.2nd selected.chromium.png|1| Changed|
|MenuList.Radio With Icons - RTL.2nd selected.chromium.png|4| Changed|
|MenuList.With Icons.hover menuitem.chromium.png|3| Changed|
|MenuList.Radio With Icons - RTL.1st selected.chromium.png|4| Changed|

</details>


  <details><summary><b>Slider</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Size Small.rightArrow.chromium.png|1| Changed|
|Slider.Size Small.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked.hover.chromium.png|1| Changed|
|Switch.Checked - RTL.hover.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>


  <details><summary><b>TextInput</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextInput.Size Large.normal.chromium.png|2| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>